docs/install.md

```diff
@@ -346,12 +344,12 @@ brew remove code-server
 brew uninstall code-server
 ```
 
-### yarn, npm
+### npm
 
 To remove the code-server global module, run:
 
 ```shell
-yarn global remove code-server
+npm -g uninstall code-server
```

Comment:
Since we're using `--global` elsewhere:

```suggestion
npm --global uninstall code-server
```

Reply:
Ah yes! Plus I actually forgot to remove the "or" section just below. Updated to `--global` + removing that section.